src/main/java/org/spongepowered/common/event/tracking/phase/packet/PacketPhase.java

```diff
@@ -93,91 +93,48 @@
-        public static final BasicInventoryPacketState SECONDARY_DRAG_INVENTORY_START = new DragInventoryStartState("SECONDARY_DRAG_INVENTORY_START", DRAG_MODE_SECONDARY_BUTTON);
-
-        public static final BasicInventoryPacketState PRIMARY_DRAG_INVENTORY_ADDSLOT = new DragInventoryAddSlotState("PRIMARY_DRAG_INVENTORY_ADD_SLOT", DRAG_MODE_PRIMARY_BUTTON);
-        public static final BasicInventoryPacketState SECONDARY_DRAG_INVENTORY_ADDSLOT = new DragInventoryAddSlotState("SECONDARY_DRAG_INVENTORY_ADD_SLOT", DRAG_MODE_SECONDARY_BUTTON);
-
-        public static final BasicInventoryPacketState PRIMARY_DRAG_INVENTORY_STOP = new PrimaryDragInventoryStopState();
-        public static final BasicInventoryPacketState SECONDARY_DRAG_INVENTORY_STOP = new SecondaryDragInventoryStopState();
-
+        public static final BasicInventoryPacketState OPEN_HORSE_INVENTORY = new IgnoredInventoryPacketState();
+        public static final BasicInventoryPacketState ENCHANT_ITEM = new IgnoredInventoryPacketState();
         public static final BasicInventoryPacketState SWITCH_HOTBAR_SCROLL = new SwitchHotbarScrollState();
-        public static final BasicInventoryPacketState OPEN_INVENTORY = new BasicInventoryPacketState();
-        public static final BasicInventoryPacketState ENCHANT_ITEM = new BasicInventoryPacketState();
-
-        static final ImmutableList<BasicInventoryPacketState> VALUES = ImmutableList.<BasicInventoryPacketState>builder()
-                .add(INVENTORY)
-                .add(PRIMARY_INVENTORY_CLICK)
-                .add(SECONDARY_INVENTORY_CLICK)
-                .add(MIDDLE_INVENTORY_CLICK)
-                .add(DROP_ITEM_OUTSIDE_WINDOW)
-                .add(DROP_ITEM_WITH_HOTKEY)
-                .add(DROP_ITEMS)
-                .add(DROP_INVENTORY)
-                .add(SWITCH_HOTBAR_NUMBER_PRESS)
-                .add(PRIMARY_INVENTORY_SHIFT_CLICK)
-                .add(SECONDARY_INVENTORY_SHIFT_CLICK)
-                .add(DOUBLE_CLICK_INVENTORY)
-
-                .add(PRIMARY_DRAG_INVENTORY_START)
-                .add(SECONDARY_DRAG_INVENTORY_START)
-
-                .add(PRIMARY_DRAG_INVENTORY_ADDSLOT)
-                .add(SECONDARY_DRAG_INVENTORY_ADDSLOT)
-
-                .add(PRIMARY_DRAG_INVENTORY_STOP)
-                .add(SECONDARY_DRAG_INVENTORY_STOP)
-
-                .add(SWITCH_HOTBAR_SCROLL)
-                .add(OPEN_INVENTORY)
-                .add(ENCHANT_ITEM)
+        public static final BasicInventoryPacketState DROP_ITEM_IN_GAME_WITH_HOTKEY = new DropItemInGameWithHotkeyState();
+        public static final BasicInventoryPacketState UNKNOWN_CLICK = new UnknownInventoryClickPacketState();
+
+        static final ImmutableList<InventoryClickPacketState> CLICK_STATES = ImmutableList.<InventoryClickPacketState>builder()
+                .add(new PrimaryInventoryClickState())
```

Comment:
Why are you creating new states instead of using the static state instances? I'm somewhat confused about this change, since it feels like you can't specifically reference the singular static state instance at this point.